commonui/src/main/java/com/hyperwallet/android/ui/common/view/HorizontalDividerItemDecorator.java

```diff
@@ -96,4 +103,9 @@ public void onDrawOver(@NonNull Canvas c, @NonNull RecyclerView parent, @NonNull
             mHorizontalItemDivider.draw(c);
         }
     }
+
+    private float getTopDividerPadding(@NonNull final Context context) {
+        return context.getResources().getDimension(R.dimen.item_horizontal_padding)
+                * context.getResources().getDisplayMetrics().density;
```

Comment:
could assign to variable to avoid calling getResources() twice. Not sure if that improves performance or not...
Resources resources = context.getResources();
resources.getDimension()...

Reply:
The method is a supplementary routine on `HorizontalDividerItemDecorator` construction that means this method is invoked once only during creation of this class, as well as during creation of the `RecyclerView` that is using this instance.

The same concept as why are we not saving `Context` context change in the lifecycle of the mobile device, that remains true with `context.Resource` so caching this objects to save routine calls is not ideal.